ballerina/constants.bal

```diff
@@ -104,6 +104,64 @@ public const int MQGMO_CONVERT = 16384;
 # Subscribe Option create
 public const int MQSO_CREATE = 2;
 
+# Encoding for normal integer representation (most significant byte first) or the big-endian format.
+public const int MQENC_INTEGER_NORMAL = 1;
```

Comment:
type descriptor is redundant for constants. any reason not to omit that?
```suggestion
public const MQENC_INTEGER_NORMAL = 1;
```

Reply:
We have used the same practice for other constants defined in the package. We will follow the same practice for time-being to maintain the consistency.